src/main/scala/org/openapitools/generator/sbt/plugin/OpenApiGeneratorPlugin.scala

```diff
@@ -45,7 +45,7 @@ object OpenApiGeneratorPlugin extends sbt.AutoPlugin
   val OpenApiCodegen = config("openApiCodegen")
 
   override def globalSettings: Seq[Def.Setting[_]] = Seq(
-    aggregate in openApiGenerators := false
+    openApiGenerators / aggregate := false
```

Comment:
can you please elaborate on this change?

looks like this caused the CI tests to fail: https://github.com/OpenAPITools/sbt-openapi-generator/actions/runs/7920353292/job/21636830291?pr=48